compiler/lib/eval.ml

```diff
@@ -330,7 +353,14 @@ let eval_instr info ((x, loc) as i) =
       | None -> [ i ])
   | Let (x, Prim (Extern "caml_sys_const_backend_type", [ _ ])) ->
       let jsoo = Code.Var.fresh () in
-      [ Let (jsoo, Constant (String "js_of_ocaml")), noloc
+      [ ( Let
+            ( jsoo
+            , Constant
+                (String
+                   (match target with
```

Comment:
I would lift this match up next to the `let jsoo = ...` above to improve overall formatting

Reply:
Done.